### PR TITLE
add dynamic page titles and rich metadata

### PR DESCRIPTION
**Pull request by kdembler**
Summary of changes:
- Add dynamic page title on each view, resolves https://github.com/Joystream/atlas/issues/510
- Add rich meta tags on channel and video views
- Use [Media Session API](https://developer.mozilla.org/en-US/docs/Web/API/Media_Session_API) on video view to provide playback metadata to the browser:<br>Screenshot 2022-02-01 at 14 37 48

### CODE REVIEW

File: packages/atlas/src/views/studio/VideoWorkspace/VideoWorkspace.tsx [hunk 1]
```diff
@@ -11,6 +12,8 @@ import { VideoWorkspaceForm } from './VideoWorkspaceForm'
 import { VideoWorkspaceTabsBar } from './VideoWorkspaceTabsBar'
 
 export const VideoWorkspace: React.FC = React.memo(() => {
+  const headTags = useHeadTags('Edit video')
```
Comment (rafalpawlow) [inline packages/atlas/src/views/studio/VideoWorkspace/VideoWorkspace.tsx]: Shouldn't we distinguish between adding end editing?

File: packages/atlas/src/views/viewer/EditMembershipView/EditMembershipView.tsx [hunk 2]
```diff
@@ -53,6 +54,8 @@ export const EditMembershipView: React.FC = () => {
     }
   }, [activeMembership, activeMembershipLoading, resetForm])
 
+  const headTags = useHeadTags('Edit membership')
```
Comment (rafalpawlow) [inline packages/atlas/src/views/viewer/EditMembershipView/EditMembershipView.tsx:57]: Same question about the distinction between editing and adding

File: packages/atlas/src/views/viewer/VideoView/VideoView.tsx [hunk 3]
```diff
@@ -108,6 +117,27 @@ export const VideoView: React.FC = () => {
...
+      return
+    }
+
+    const artwork: MediaImage[] = thumbnailUrl ? [{ src: thumbnailUrl, type: 'image/webp', sizes: '640x360' }] : []
+
+    navigator.mediaSession.metadata = new MediaMetadata({
+      title: video.title || '',
+      artist: video.channel.title || '',
+      album: '',
+      artwork: artwork,
```
Comment (rafalpawlow) [inline packages/atlas/src/views/viewer/VideoView/VideoView.tsx:133]:
```suggestion
      artwork,
```

**Review by rafalpawlow — CHANGES_REQUESTED** [review]

File: packages/atlas/src/views/viewer/EditMembershipView/EditMembershipView.tsx (hunk 2 above)
Comment (kdembler) [reply packages/atlas/src/views/viewer/EditMembershipView/EditMembershipView.tsx:57]: Here you can only edit

File: packages/atlas/src/views/studio/VideoWorkspace/VideoWorkspace.tsx (hunk 1 above)
Comment (kdembler) [reply packages/atlas/src/views/studio/VideoWorkspace/VideoWorkspace.tsx]: Wanted to keep it simple, but updated now, also for channel edit

**Review by rafalpawlow — APPROVED** [review]

**Pull request merged**